samples/bluetooth/direction_finding_central/sample.yaml

```diff
@@ -10,7 +10,7 @@ tests:
         - nrf52833dk_nrf52833
         - nrf52833dk_nrf52820
         - nrf5340dk_nrf5340_cpuapp
-  sample.bluetooth.direction_finding_central.aod:
+  sample. bluetooth.direction_finding_central.aod:
```

Comment:
The change cannot be just an extra white space

Reply:
Done